packages/editor/src/components/document-bar/index.js

```diff
@@ -184,11 +187,19 @@ export default function DocumentBar( props ) {
 									? decodeEntities( title )
 									: __( 'No title' ) }
 							</span>
-							{ postTypeLabel && ! props.title && (
+							{ pageTypeBadge && ! props.title && (
 								<span className="editor-document-bar__post-type-label">
-									{ '· ' + decodeEntities( postTypeLabel ) }
+									{ '. ' + pageTypeBadge }
 								</span>
 							) }
+							{ postTypeLabel &&
+								! props.title &&
+								! pageTypeBadge && (
+									<span className="editor-document-bar__post-type-label">
```

Comment:
Would it be more readable to move the conditional inside the `<span>` tag since it's the same for both `conditions`

```suggestion
<span className="editor-document-bar__post-type-label">
    { '. ' + ( postTypeLabel && ! props.title && ! pageTypeBadge ) ?  decodeEntities( postTypeLabel ) : pageTypeBadge }
</span>
```

Reply:
Thanks @Vrishabhsk, Updated the PR with the same.

Reply:
I would recommend to avoid to potentially render an empty span element so I'd keep the condition wrapping the span.